ndcube/ndcube.py

```diff
@@ -255,20 +259,33 @@ def dimensions(self):
         Returns a named tuple with two attributes: 'shape' gives the shape
         of the data dimensions; 'axis_types' gives the WCS axis type of each dimension,
         e.g. WAVE or HPLT-TAN for wavelength of helioprojected latitude.
+
+        """
+        return u.Quantity(self.data.shape, unit=u.pix)
+
+    @property
+    def world_axis_physical_types(self):
         """
+        Returns an iterable of strings describing the physical type for each world axis.
+
+        They should be names from the VO UCD1+ controlled
+        Vocabulary (http://www.ivoa.net/documents/latest/UCDlist.html).
+        If no matching UCD type exists, this can instead be "custom:xxx",
+        where xxx is an arbitrary string.  Alternatively, if the physical
+        type is unknown/undefined, an element can be `None`.
 
+        """
         ctype = list(self.wcs.wcs.ctype)
         axes_ctype = []
         for i, axis in enumerate(self.missing_axis):
             if not axis:
-                axes_ctype.append(ctype[i])
-        shape = u.Quantity(self.data.shape, unit=u.pix)
-        return DimensionPair(shape=shape, axis_types=axes_ctype[::-1])
+                axes_ctype.append(wcs_ivoa_mapping[ctype[i]])
```

Comment:
You should probably use `.get` here to default to `None` if we don't know what it is.

Reply:
Good idea!  But what about the default being ```custom:unknown```?  That way it's always a string and has a similar format?

Reply:
The APE spec says it should be None if it's undefined.

Reply:
Fair enough.